Docs/source/building/openpmd.rst

```diff
@@ -16,74 +16,90 @@ library from source.
 
 In order to install spack, you can simply do:
 
-::
+.. code-block:: bash
 
-  git clone https://github.com/spack/spack.git
-  export SPACK_ROOT=/path/to/spack
-  . $SPACK_ROOT/share/spack/setup-env.sh
+   git clone https://github.com/spack/spack.git
+   export SPACK_ROOT=/path/to/spack  # note: adjust this path
```

Comment:
```suggestion
   export SPACK_ROOT=</path/to>/spack  # note: adjust the path to point to your spack repo location
```

Reply:
Good idea!
I am not 100% happy with the line since in reality it will be `$PWD/spack` at this point but in order to add it to `.bashrc`/`.bash_profile` configs (next sentence) one needs the absolute path again... let me add another idea.

Reply:
Ah that makes sense yes!